src/docz/Tooltip.mdx

```diff
@@ -0,0 +1,82 @@
+---
+name: Tooltip
+route: /tooltip
+---
+
+import Tooltip from '../core/Tooltip'
+import Button from '../core/Button'
+import { Playground, PropsTable } from 'docz'
+import { Add as IconAdd } from '../icons'
+
+# Tooltip
+
+## Properties
+<PropsTable of={ Tooltip } />
+
+## Simple tooltip
+<Playground>
+    <Tooltip title='Tooltip'>
+        <Button variant='outlined'>Simple tooltip</Button>
+    </Tooltip>
+</Playground>
+
+## Positioned tooltip
+<Playground>
+    <Tooltip title='Tooltip' placement='left-start'>
+        <Button variant='outlined' color='secondary' margin={ 12 }>left-start</Button>
+    </Tooltip>
+    <Tooltip title='Tooltip' placement='left'>
+        <Button variant='outlined' color='primary' margin={ 12 }>left</Button>
+    </Tooltip>
+    <Tooltip title='Tooltip' placement='left-end'>
+        <Button variant='outlined' margin={ 12 }>left-end</Button>
+    </Tooltip>
+    <Tooltip title='Tooltip' placement='bottom-start'>
+        <Button variant='outlined' color='primary' margin={ 12 }>bottom-start</Button>
+    </Tooltip>
+    <Tooltip title='Tooltip' placement='bottom'>
+        <Button variant='outlined' color='secondary' margin={ 12 }>bottom</Button>
+    </Tooltip>
+    <Tooltip title='Tooltip' placement='bottom-end'>
+        <Button variant='outlined' color='primary' margin={ 12 }>bottom-end</Button>
+    </Tooltip>
+    <Tooltip title='Tooltip' placement='right-start'>
+        <Button variant='outlined' margin={ 12 }>right-start</Button>
+    </Tooltip>
+    <Tooltip title='Tooltip' placement='right'>
+        <Button variant='outlined' color='primary' margin={ 12 }>right</Button>
+    </Tooltip>
+    <Tooltip title='Tooltip' placement='right-end'>
+        <Button variant='outlined' color='secondary' margin={ 12 }>right-end</Button>
+    </Tooltip>
+    <Tooltip title='Tooltip' placement='top-start'>
+        <Button variant='outlined' margin={ 12 }>top-start</Button>
+    </Tooltip>
+    <Tooltip title='Tooltip' placement='top'>
+        <Button variant='outlined' color='secondary' margin={ 12 }>top</Button>
+    </Tooltip>
+    <Tooltip title='Tooltip' placement='top-end'>
+        <Button variant='outlined' color='primary' margin={ 12 }>top-end</Button>
+    </Tooltip>
+</Playground>
+
+## Controlled tooltip
+<Playground>
+    <Tooltip
+        title='Tooltip'
+        onClose={ ()=> console.log('onClose()') }
+        onOpen={ ()=> console.log('onOpen()') }
+        open={ true }>
```

Comment:
`open` is a shorthand property, it means you can do:

```js
<Tooltip
    open
    title='Tooltip'
    onClose={ () => console.log('onClose()') }
    onOpen={ () => console.log('onOpen()') }>
```

Reply:
Solved